docs/specifications/tests/TestMessages.md

```diff
@@ -11,8 +11,12 @@ Index of Text Cases are found in [README](README.md).
 | DOMAIN_NAME_LABEL_TOO_LONG               | [Basic](Basic-TP/README.md) | [basic00](Basic-TP/basic00.md) |
 | DOMAIN_NAME_TOO_LONG                     | [Basic](Basic-TP/README.md) | [basic00](Basic-TP/basic00.md) |
 | DOMAIN_NAME_ZERO_LENGTH_LABEL            | [Basic](Basic-TP/README.md) | [basic00](Basic-TP/basic00.md) |
+| TEST_CASE_END                            | [Basic](Basic-TP/README.md) | [basic00](Basic-TP/basic00.md) |
```

Comment:
TEST_CASE_START and TEST_CASE_END message are mapped to multiple test cases. Is it needed to map both the messages  to test cases as they are generic?

Reply:
It is a good point. The document is generated by a script. I could manually remove those lines for every test case for this release. Do you want me to do that?

For upcoming releases we could either handle those specially in the code so that they do not show in TestMessages.md or update the script to make an exception to those messages.

Reply:
We could do that for the upcoming release.

Reply:
Another aspect of these messages is that they should not be translated, so it would be better if they do not appear in the generated PO files.